### DISCUSSION TITLE
How to enable Agama ?

### DISCUSSION THREAD

**Discussion by toroarnaud** [JanssenProject/jans#6568] [Q&A] [answered]
Hi,

I have a janssen-auth server up and running on Kubernetes.
I have created an Agama project in the very nice Agama lab and I about to deploy it using TUI.
The issue is that when selecting _Upload projects_ a message pops up with "_Action must be taken / Agama is not enabled!_".
The message is pretty clear but I can't find anywhere how to enable it. I had a look in the values.yaml reference file in the Janssen project and I could not see any _agama_ section there.
I would appreciate if someone could guide me on this...

Thank you,
Arnaud

**Comment by syntrydy** [comment, marked as answer]
The process  is defined here https://docs.jans.io/head/admin/developer/agama/engine-bridge-config/#availability

**Reply by toroarnaud** [reply]
Ooops! I had missed it... Thank you very much!
